### DISCUSSION TITLE
Home Assistant in 2 abitazioni

### DISCUSSION THREAD

**Discussion by Ale1975-Iron** [indomus/forum#511] [HUB: Home Assistant]
Ciao a tutti,

potrebbe essere una domanda banale ma non ho trovato info in merito.
Ipotizzando di voler domotizzare con home assistant su 2 raspberry in 2 abitazioni che utilizzano 2 differenti Wifi. La mia domanda è 
Posso fare monitoraggio di entrambe dall’applicazione del telefono?

Ciao e Grazie

**Comment by maxcanna** [comment]
No, purtroppo l'applicazione non permette l'aggiunta di installazioni multiple. Puoi però ovviare salvando sulla home del telefono due collegamenti ad entrambe le installazioni. L'interazione ed il risultato finale saranno pressoché identiche

**Reply by Ale1975-Iron** [reply]
In effetti un’ottima e semplice soluzione a cui non avevo neppure pensato. Grazie mille per la dritta

**Comment by KC-inDomus** [comment]
Puoi integrare un'istanza sull'altra e controllare entrambe da una sola app. Puoi usare questo componente https://github.com/custom-components/remote_homeassistant

**Reply by Ale1975-Iron** [reply]
Grazie per l’alternativa 😃
